### PR TITLE
DESCW-975 Updated Block Theme documentation

### PR DESCRIPTION
**Pull request by codewisenate**
Updated instructions and documentation include graphics where appropriate. Additions to the existing documentation include: 
- Using custom block features such as Accessibility labels and print image option. 
- Fluid typography and responsive margins and padding. 
- Creating custom patterns (and how to enable it). 
- Using the custom notification banner (and how to enable it). 
- Multi-language supported placement and settings for: 
  - adding Polylang language switcher (to the menu or header)
  - translation of FSE template blocks and decoupling Template Parts. 
- Adding a Custom Security Policy exception.

### CODE REVIEW

File: src/Templates/InfoAdminPage.php [hunk 1]
```diff
@@ -300,17 +417,45 @@ public function build_options_page() {
...
+
+							<p><img src="<?php echo esc_url( get_template_directory_uri() ) . '/assets/images/block-theme/notice-settings.png'; ?>" alt="Settings for the custom notice banner" style="border: 1px solid #ccc; width: 100%;" /><br />Settings for the custom notice banner</p>
+
+							<ol>
+							<li> <strong>Enable Admin Notification:</strong> Show or hide the banner on the site.</li>
+							<li> <strong>Homepage only:</strong> This refers to a setting that allows the banner to be displayed only on the homepage and not on other pages.</li>
+							<li> <strong>Status (colour):</strong> The colour used to indicate the status of the message. A number of colour options are available (which can be seen in the image above) and utilise preset colour options that conform to digital accessibility standards for contrast.</li>
```
Comment (ShawnTurple) [inline src/Templates/InfoAdminPage.php:439]:
```suggestion
							<li> <strong>Status (colour):</strong> The colour used to indicate the status of the message. A number of colour options are available (which can be seen in the image above) and utilize preset colour options that conform to digital accessibility standards for contrast.</li>
```

File: src/Templates/InfoAdminPage.php [hunk 2]
```diff
@@ -300,17 +417,45 @@ public function build_options_page() {
...
+							<li> <strong>Embolden / Embiggen:</strong> These terms refer to the act of making the banner content appear more prominent by increasing the font size or bolding the text.</li>
+							<li> <strong>Notification content:</strong> This refers to the message or information displayed in the notification banner itself, such as an alert or banner, that is meant to grab the attention of the user.</li>
+							<li> <strong>Enable button:</strong> Show or hide the button inside the banner.</li>
+							<li> <strong>Button Label:</strong> This is the text that appears inside the button.</li>
+							<li> <strong>Accessibility button label:</strong> This is the text that is read by screen readers to describe the purpose or function of a button to users with disabilities. It is important to ensure that this label accurately reflects the button's purpose to ensure accessibility.</li>
+							<li> <strong>UTM Campaign:</strong> A UTM campaign is a set of parameters added to a URL to track the effectiveness of online marketing campaigns. If the button is meant to track click throughs to a specific campaign, add its UTM identifier here. It is ok to leave this empty.</li>
```
Comment (ShawnTurple) [inline src/Templates/InfoAdminPage.php:445]:
```suggestion
							<li> <strong>UTM Campaign:</strong> A UTM campaign is a set of parameters added to a URL to track the effectiveness of online marketing campaigns. If the button is meant to track click-through to a specific campaign, add its UTM identifier here. It is ok to leave this empty.</li>
```

File: src/Templates/InfoAdminPage.php [hunk 3]
```diff
@@ -219,21 +303,21 @@ public function build_options_page() {
...
 
 							<p><img src="<?php echo esc_url( get_template_directory_uri() ) . '/assets/images/block-theme/BCGOV-Colour-Palette-Swatches.jpg'; ?>" alt="BCGov theme colour palette" style="width: 300px;" /> <img src="<?php echo esc_url( get_template_directory_uri() ) . '/assets/images/block-theme/BCGOV-Gradients-Swatches.jpg'; ?>" alt="BCGov theme colour palette" style="width: 300px;" /></p>
 							
 							<p><img src="<?php echo esc_url( get_template_directory_uri() ) . '/assets/images/block-theme/CleanBC-Colour-Palette-Swatches.jpg'; ?>" alt="CleanBC theme colour palette" style="width: 300px;" /> <img src="<?php echo esc_url( get_template_directory_uri() ) . '/assets/images/block-theme/CleanBC-Gradients-Swatches.jpg'; ?>" alt="BCGov theme colour palette" style="width: 300px;" /></p>
 							
-							<p>It should be noted that most blocks and patterns can be modified in the editor to override colour or gradient palette options to accomodate specific customisation needs.</p>
+							<p>It should be noted that most blocks and patterns can be modified in the editor to override colour or gradient palette options to accommodate specific customisation needs.</p>
```
Comment (ShawnTurple) [inline src/Templates/InfoAdminPage.php:314]:
```suggestion
							<p>It should be noted that most blocks and patterns can be modified in the editor to override colour or gradient palette options to accommodate specific customization needs.</p>
```

File: src/Templates/InfoAdminPage.php [hunk 4]
```diff
@@ -276,7 +393,7 @@ public function build_options_page() {
...
+							<p>The theme editor allows for the customization of the BCGov WordPress theme without writing any code, and modifying page templates without breaking the website. This page is not meant for in-depth instruction in using the theme editor, just know the full site editor is a powerful and flexible way to customise the BCGov Block theme and will allow for making substantive changes to a theme and its layout at any time. This enables a level of flexibility which allows you to easily create on-demand page layouts. For more information regarding WordPress 6+ Templating see: </p>
 
 						</div><!-- /Customisable Templates -->
```
Comment (ShawnTurple) [inline src/Templates/InfoAdminPage.php:398]:
```suggestion
						</div><!-- /Customizable Templates -->
```

File: src/Templates/InfoAdminPage.php [hunk 5]
```diff
@@ -300,17 +417,45 @@ public function build_options_page() {
 
 							<h2>Instructions</h2>
 
-							<p>Template parts are used to organize a theme in smaller reusable structural parts. They are commonly used for site headers and site footers. Just like templates, the theme editor allows for the customization of the BCGov WordPress theme without writing any code and this includes modifying template parts.</p>
+							<p>Template parts are used to organise a theme in smaller reusable structural parts. They are commonly used for site headers and site footers. Just like templates, the theme editor allows for the customization of the BCGov WordPress theme without writing any code and this includes modifying template parts.</p>
 
 							<p>The Site Editor’s Template Parts section displays a list of all template parts. You can create template parts using the Add New button. In this view, you can also clear customizations from theme template parts and delete user-created template parts should an error be made while editing inside the editor.</p>
 
 						</div><!-- /Customisable Template Parts -->
```
Comment (ShawnTurple) [inline src/Templates/InfoAdminPage.php:424]:
```suggestion
						</div><!-- /Customizable Template Parts -->
```

**Review by ShawnTurple — CHANGES_REQUESTED** [review]

**Review by ShawnTurple — APPROVED** [review]

File: src/scripts/admin/blocks/card/container/deprecated.js [hunk 6]
```diff
@@ -1,3 +1,5 @@
+/* eslint react/no-unknown-property: "off" */
```
Comment (Baelx) [inline src/scripts/admin/blocks/card/container/deprecated.js:1]: Can I ask why this was necessary? Were you trying to use a custom property/attribute on an element?

**Pull request merged**